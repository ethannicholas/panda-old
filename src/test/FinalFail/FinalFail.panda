class FinalFail {
    @final
    var x:Int

    constructor() {
        x := 1239
        x := 5920
    }
    
    @class
    method main() {
        var final := new FinalFail()
        Console.writeLine(final.x)
    }
}