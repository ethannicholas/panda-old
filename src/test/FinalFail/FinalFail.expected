error: FinalFail.panda:7:11: cannot modify final field 'x'
1 error
